v2/get_catalog_test.go

```diff
@@ -51,9 +51,9 @@ func okCatalogResponse() *CatalogResponse {
 				Requires: []string{
 					"route_forwarding",
 				},
-				Bindable:      true,
+				Bindable:           true,
```

Comment:
Ignore this file; formatting changes from `gofmt` related to previous PR.

Reply:
Yeah, we need a verify gofmt script to run on this repo.